WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitsBanner.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+
+/// Hosting controller wrapper for `JetpackBenefitsBanner`
+///
+final class JetpackBenefitsBannerHostingController: UIHostingController<JetpackBenefitsBanner> {
+    init() {
+        super.init(rootView: JetpackBenefitsBanner())
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    /// Actions are set in a separate function because most of the time, they will require to access `self` to be able to present new view controllers.
+    ///
+    func setActions(tapAction: @escaping () -> Void, dismissAction: @escaping () -> Void) {
+        rootView.tapAction = tapAction
+        rootView.dismissAction = dismissAction
+    }
+}
+
+/// A banner about Jetpack benefits that can be tapped to show more details or dismiss.
+struct JetpackBenefitsBanner: View {
+    /// Closure invoked when the banner is tapped
+    ///
+    var tapAction: () -> Void = {}
+
+    /// Closure invoked when the dismiss button is tapped
+    ///
+    var dismissAction: () -> Void = {}
+
+    // Tracks the scale of the view due to accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        Group {
+            HStack(spacing: Layout.horizontalSpacing) {
+                Image("icon-jetpack-gray")
```

Comment:
nit: can we use the existing `jetpackLogoImage` variable under `UIImage` extension?

Reply:
Great catch! Updated in https://github.com/woocommerce/woocommerce-ios/pull/5378/commits/5020fa4a894423cd617dd030ea8c8aa3794e961d